src/OrbitGl/TimelineTicks.h

```diff
@@ -29,7 +29,8 @@ class TimelineTicks {
   [[nodiscard]] std::vector<uint64_t> GetMajorTicks(uint64_t start_ns, uint64_t end_ns) const;
 
  private:
-  uint64_t GetScale(uint64_t time_range_ns) const;
+  [[nodiscard]] uint64_t GetScale(uint64_t time_range_ns) const;
+  [[nodiscard]] uint64_t GetPreviousScale(uint64_t tick_scale) const;
```

Comment:
Nit: The name of this method is ambiguous, what's "Previous"? I would call this something like `GetMinorTicksScale`, or `GetMorePreciseScale`

Reply:
GetMinorTicksScale and GetMajorTicksScale